docs/content/developer/iota-identity/how-tos/post-quantum.mdx

```diff
@@ -0,0 +1,422 @@
+        }
+      }
+    ]
+  },
+  "meta": {
+    "created": "2025-09-13T12:49:03Z",
+    "updated": "2025-09-13T12:49:03Z"
+  }
+}
+
+```
+
+As expected we see a DID verification method of type `JsonWebKey2020` encapsulating an ML-DSA-87
+public key with all its outstanding 2592 bytes!
+
+For signing a VC with a PQ key, users can invoke the `create_credential_jwt_pqc` on their DID
+document, passing to it a reference to their key store and the DID verification method's fragment
+they want to use.
+
+<Tabs groupId="language" queryString>
+<TabItem value="typescript-node" label="Typescript (Node.js)">
+
+```ts 
+const credentialJwt = await issuerDocument.createCredentialJwtPqc(
+  issuerStorage,
+  issuerFragment,
+  unsignedVc,
+  new JwsSignatureOptions(),
+);
+```
+
+</TabItem>
+<TabItem value="rust" label="Rust">
+
+```rust
+let credential_jwt = issuer_document
+  .create_credential_jwt_pqc(
+    &credential,
+    &issuer_storage,
+    &issuer_fragment,
+    &JwsSignatureOptions::default(),
+    None,
+  )
+  .await?;
+```
+
+</TabItem>
+</Tabs>
+
+The result of this operation is a JWT encoded VC, with an algorithm (`alg` property) matching that
+of the verification method used, and a particularly long signature: 4627 bytes when using ML-DSA-87.
+
+To verify the signature on a JWT-encoded VC or VP that is using a PQ algorithm, the `PQCJwsVerifier`
+must be used. Using a JWT-encoded VC as an example:
+
+<Tabs groupId="language" queryString>
+<TabItem value="typescript-node" label="Typescript (Node.js)">
+
+```ts 
+new JwtCredentialValidator(new PQJwsVerifier()).validate(
+  credentialJwt,
+  issuerDocument,
+  new JwtCredentialValidationOptions(),
+  FailFast.FirstError,
+);
+```
+
+</TabItem>
+<TabItem value="rust" label="Rust">
+
+```rust
+JwtCredentialValidator::with_signature_verifier(PQCJwsVerifier::default())
+  .validate(
+    &credential_jwt,
+    &issuer_document,
+    &JwtCredentialValidationOptions::default(),
+    FailFast::FirstError,
+  )?;
+```
+
+</TabItem>
+</Tabs>
+
+## Transitioning to Post-Quantum Cryptography: Hybrid Signatures
+
+Hybrid signatures can ease the transition to post-quantum cryptography (PQC) by providing a "best of
```

Comment:
We should expand on how such a transition would look like (phase out pre-quantum verification I'd say).
Not critical for the initial version though.